src/WorkItems/TestWorkItemData.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.WorkItems
+{
+    public class TestWorkItemData
```

Comment:
>TestWorkItemData [](start = 17, length = 16)

Shouldn't this be in a test assembly rather than a product code assembly?

Reply:
VS auto-created this in response to some refactoring misfire I completed.

---
In reply to: [381448969](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381448969) [](ancestors = 381448969)